enterprise/frontend/src/embedding-sdk/cli/snippets/embedding-provider-snippet.ts

```diff
@@ -39,6 +39,7 @@ const authConfig = {
 // Demo provider that wraps the MetabaseProvider with a custom theme and auth configuration.
 // In a real app, the theme would be managed by your application.
 export const EmbeddingProvider = ({children}) => {
+  /** @type {import('./analytics-provider').AnalyticsContextType} */
```

Comment:
I think this would already be inferred from the `AnalyticsContext` object as you already typed the whole `AnalyticsContext` via `@type` in that file